packages/flutter_tools/lib/src/observatory.dart

```diff
@@ -74,6 +74,11 @@ class Observatory {
     _getEventController(data['streamId']).add(event);
   }
 
+  void populateIsolateInfo() {
+    // Calling this has the side effect of populating the isolate information.
+    waitFirstIsolate;
```

Comment:
What does this do?  Is this a function call somehow?  This is just the first usage of a final variable?